src/DataFramesMeta.jl

```diff
@@ -43,27 +46,39 @@ replace_syms!(e::Expr, membernames) =
 
 protect_replace_syms!(e, membernames) = e
 protect_replace_syms!(e::Expr, membernames) =
-    e.head == :quote ? e : replace_syms!(e, membernames)
+    if e.head == :quote
```

Comment:
I wouldn't say that's a sufficient reason. If think it's been discussed in Base already.

Anyway, when you use `if` blocks, please also use the long `function` form (there are several cases like this), and keep the short syntax for one-liners.